seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/config/KuduSinkConfig.java

```diff
@@ -26,11 +29,25 @@
 @Data
 public class KuduSinkConfig {
 
-    private static final String KUDU_SAVE_MODE = "save_mode";
-    private static final String KUDU_MASTER = "kudu_master";
-    private static final String KUDU_TABLE_NAME = "kudu_table";
+    public static final Option<String> KUDU_MASTER =
+        Options.key("kudu_master")
+            .stringType()
+            .noDefaultValue()
+            .withDescription("kudu master address");
+
+    public static final Option<SaveMode> KUDU_SAVE_MODE =
+        Options.key("save_mode")
+            .enumType(SaveMode.class)
+            .noDefaultValue()
```

Comment:
set default value to `APPEND`?

Reply:
At present, this parameter is defined as mandatory. If necessary, I can change it to non mandatory and add the default value